store/account/account_test.go

```diff
@@ -62,48 +65,49 @@ var _ = Describe("Account Database Test", func() {
 		Expect(err).Should(Succeed())
 		Expect(mysql.Start()).Should(Succeed())
 
-		db, err = gorm.Open("mysql", mysql.URL)
+		db, err = sqldb.SimpleConnect("mysql", mysql.URL)
 		Expect(err).Should(Succeed())
 		Expect(db).ShouldNot(BeNil())
-
-		db.LogMode(os.Getenv("ENABLE_DB_LOG_IN_TEST") != "")
 	})
 
 	AfterSuite(func() {
 		mysql.Stop()
 	})
 
 	BeforeEach(func() {
-		db.Delete(&model.Header{})
-
 		// Drop erc20 contract table
-		codes := []model.ERC20{}
-		db.Find(&codes)
+		store := NewWithDB(db)
+		codes, err := store.ListERC20(ctx)
+		Expect(err).Should(BeNil())
 		for _, code := range codes {
-			db.DropTable(model.Transfer{
+			_, err := db.Exec(fmt.Sprintf("DROP TABLE %s", model.Transfer{
 				Address: code.Address,
-			})
-			db.DropTable(model.Account{
+			}.TableName()))
+			Expect(err).Should(BeNil())
+			_, err = db.Exec(fmt.Sprintf("DROP TABLE %s", model.Account{
 				ContractAddress: code.Address,
-			})
+			}.TableName()))
+			Expect(err).Should(BeNil())
 		}
 
-		db.Delete(&model.ERC20{})
-		db.Delete(&model.Account{})
-		db.Delete(&model.Account{
-			ContractAddress: model.ETHBytes,
-		})
+		_, err = db.Exec("DELETE FROM erc20")
+		Expect(err).Should(Succeed())
+		// Remove ETH accounts & balances
+		_, err = db.Exec("DELETE FROM accounts")
+		Expect(err).Should(Succeed())
+		_, err = db.Exec("DELETE FROM eth_transfer")
+		Expect(err).Should(Succeed())
```

Comment:
why didn't we have to delete any erc20 table?

Reply:
We drop them.
https://github.com/getamis/eth-indexer/pull/100/files/68dc4ecdab129115797ce73ffb7093dbe0a7b0a6..c60c147d5edf8262bae0dae91ef3172715ffd9cd#diff-1d2f602ad7e8a4ec7bfadb89f31ba418R87